feat(docs): Writing tutorials (#157)
kestra-io/kestra.io: docs/tutorial/outputs/README.md

---
order: 3
---

# Outputs

Each task can generate output data. And those data are passed to the following Tasks. These outputs can be variables or files stored inside Kestra's [internal storage](../../architecture#storage).

## Accessing outputs

> Task output can be accessed during the flow using the <code v-pre>{{ outputs['task-id'] }}</code> syntax. If your task's id contains no `-`, you can directly use <code v-pre>{{ outputs.taskId }}</code>.

Reviewer: ```suggestion
Task outputs can be accessed inside the flow using the <code v-pre>{{ outputs['task-id'] }}</code> syntax. If your task's id contains no `-`, you can also use the <code v-pre>{{ outputs.taskId }}</code> syntax.
```
